util/reggen/multi_register.py

```diff
@@ -69,6 +69,7 @@ def __init__(self,
         self.cname = check_name(rd['cname'],
                                 'cname field of multireg {}'
                                 .format(self.reg.name))
+        self.name = self.cname
```

Comment:
I think this should probably be `self.reg.name`, not `self.cname`. If defined, the `cname` field seems to be used for the name of the component and is the same for multiple multiregs. See `flash_ctrl.hjson`, for example.

Reply:
Thanks for pointing this out. Fixed.